Tella/Scenes/Nextcloud/ViewModels/NextcloudMainViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  GDriveMainViewModel.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 2/7/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
```

Comment:
I think no need for importing Foundation

Reply:
I think we need `import Foundation` for `DispatchQueue`